composer.lock

```diff
@@ -1,6933 +0,0 @@
-{
```

Comment:
The file should still be in the source code.

Reply:
Yes, I'll add this later. :ok_hand:

Reply:
After adding stable version of yii2-apidoc.

Reply:
Added for the latest commit in master. A tag needs to be created and I'll update this.

Reply:
:heavy_check_mark: